core/trino-main/src/test/java/io/trino/server/security/TestResourceSecurity.java

```diff
@@ -802,7 +802,21 @@ public TestResource(AccessControl accessControl)
 
         @ResourceSecurity(AUTHENTICATED_USER)
         @GET
-        public javax.ws.rs.core.Response echoToken(@Context HttpServletRequest servletRequest, @Context HttpHeaders httpHeaders)
+        @javax.ws.rs.Path("/protocol/identity")
```

Comment:
Why not to use `/ui/api/identity` for both case?

Reply:
`/ui/api/identity` uses Web UI authentication filter (`OAuth2WebUiAuthenticationFilter`) whereas `/protocol/identity`  uses protocol authenticator (`OAuth2Authenticator`).